doc/dev/cephfs-mirroring.rst

```diff
@@ -287,14 +309,20 @@ E.g::
     }
   ]
 
-An entry per mirror daemon instance is displayed along with information such as configured
-peers and basic stats. For more detailed stats, use the admin socket interface as detailed
-below.
+One entry per mirror-daemon instance is displayed, along with information
+including configured peers and basic statistics. For more detailed statistics,
+use the admin socket interface as detailed below.
+
+CephFS mirror daemons provide admin socket commands for querying mirror status.
```

Comment:
s/provide/listen for/

Reply:
I'll let the experts chime in on this and then make another pass at it in about eighteen hours. I appreciate the hard eye cast upon this flabby text.

Reply:
Provide does work too.